@utahdts/utah-design-system/react/components/popups/Modal/ModalTitle.jsx

```diff
@@ -4,15 +4,18 @@ import { joinClassNames } from '../../../util/joinClassNames';
  * @param {object} props
  * @param {import('react').ReactNode} props.children
  * @param {string} [props.className]
+ * @param {string} props.id
```

Comment:
does `id` need to be required?

Reply:
Maybe add a comment in the jsdoc to explain it